tests/run_tests.py

```diff
@@ -105,15 +115,17 @@ def run_ctests(self):
 
 
 # Simple function to add a test to the global tests dictionary
-def add_test(name, num_threads=1, debug=False, ):
-    tests.update({name: Test(name, num_threads, debug)})
+def add_test(name, cc='gcc', num_threads=1, debug=False, ):
+    tests.update({name: Test(name, cc, num_threads, debug)})
 
 # List of all tests that may be run. User can add -C to command line to specify
 # a subset of these configurations
-add_test('normal', num_threads=1)
-#add_test('debug', num_threads=1, debug=True)
-#add_test('normal-openmp', num_threads=4)
-#add_test('debug-openmp', num_threads=4, debug=True)
+add_test('normal-gcc', cc='gcc', num_threads=1)
+add_test('normal-openmp-gcc', cc='gcc', num_threads=4)
+#add_test('normal-icpc', cc='icpc', num_threads=1)
+#add_test('normal-openmp-icpc', cc='icpc', num_threads=4)
+#add_test('normal-clang', cc='clang', num_threads=1)
+#add_test('normal-openmp-clang', cc='clang', num_threads=4)
```

Comment:
Yes, at the moment it is not easy to install Intel compilers on Travis but this may change soon (see [this](https://github.com/travis-ci/travis-ci/issues/4604) discussion). The version of `clang` installed by `apt-get` on Ubuntu 12.04 is not new enough to support some of the C++11 features we need. However, if you want to run the test suite with `icpc` or `clang` on a personal machine or nsecluster, all you have to do is uncomment the appropriate lines here. So the reason for these lines is simply to show them as an example.

I will say that the test suite passes on my machine for `clang`, but fails for some tests with `icpc` on nsecluster. It appears that a few fluxes for some tests vary in the 6th decimal place in sequential mode, and many more vary when run in parallel, indicating that roundoff error is different with Intel than GNU.